<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShopGoodsDialog</class>
 <widget class="QWidget" name="ShopGoodsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>196</width>
    <height>170</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Goods</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_ok">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QRadioButton" name="radio_armor">
       <property name="text">
        <string>Armor</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="radio_weapon">
       <property name="text">
        <string>Weapon</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="radio_item">
       <property name="text">
        <string>Item</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="combo_item">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="combo_weapon">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="combo_armor">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radio_item</tabstop>
  <tabstop>combo_item</tabstop>
  <tabstop>radio_weapon</tabstop>
  <tabstop>combo_weapon</tabstop>
  <tabstop>radio_armor</tabstop>
  <tabstop>combo_armor</tabstop>
  <tabstop>button_ok</tabstop>
  <tabstop>button_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radio_item</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_item</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>91</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_weapon</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_weapon</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_armor</sender>
   <signal>toggled(bool)</signal>
   <receiver>combo_armor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
